kdl-client/src/Pages/Server/pages/Server/components/ServerBar/components/Breadcrumbs/components/Crumb/Crumb.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useEffect, useRef, useState } from 'react';
+import styles from './Crumb.module.scss';
+import AnimateHeight from 'react-animate-height';
+import { useClickOutside } from 'kwc';
+import cx from 'classnames';
+
+export type CrumbProps = {
+  crumbText: string;
+  LeftIconComponent: any;
+  RightIconComponent?: any;
+  bottomComponent: React.ReactElement;
+};
+
+function Crumb({
+  crumbText,
+  LeftIconComponent,
+  RightIconComponent,
+  bottomComponent,
+}: CrumbProps) {
+  const crumbReference = useRef(null);
+  const [showComponent, setShowComponent] = useState(false);
+  const { addClickOutsideEvents, removeClickOutsideEvents } = useClickOutside({
+    componentRef: crumbReference,
+    action: () => setShowComponent(false),
+  });
+
+  useEffect(() => {
+    if (crumbReference && showComponent) addClickOutsideEvents();
+    else removeClickOutsideEvents();
+  }, [showComponent, addClickOutsideEvents, removeClickOutsideEvents]);
+
+  return (
+    <div
+      className={styles.container}
+      onClick={() => setShowComponent(!showComponent)}
+      ref={crumbReference}
+    >
+      <LeftIconComponent className={'icon-regular'} />
```

Comment:
when adding string values that do not depend on variables, do not use brackets:
```suggestion
      <LeftIconComponent className="icon-regular" />
```

I will take a look, I think I can add this check to the linter.